wistiavideo/static/html/wistiavideo.html

```diff
@@ -0,0 +1,9 @@
+<div class="wistiavideo_block">
+  <script src="//fast.wistia.com/embed/medias/{self.media_id}.jsonp" async></script>
```

Comment:
There are two types of embed codes. Async and iframe. This xblock uses async, which is required to work with the player's js api. Since we're planning to add more features like subtitles navigation and replace wistia's player controls with the ones from Open edX, this is the way to go.

By default player relies on html5 and falls back to Flash if browser does not support it. The player does it transparently.

Also, player supports following browsers ([docs](https://wistia.com/doc/viewer-requirements#browsers_supported)):
* All of the major browsers and versions of Internet Explorer (as far back as IE8), Firefox, Safari, and Chrome.
* All iOS devices, and Android devices running version 2.2 or later.